### PR TITLE
Update GA Roadmap

### PR DESCRIPTION
**Pull request by stefanprodan**
Updates:
- Added a new section for Git improvements
- Added link to RFC-0003
- Added cosgin to the OCI section

Preview: https://deploy-preview-1094--fluxcd.netlify.app/roadmap/

### CODE REVIEW

**Review by makkes — APPROVED** [review]

File: content/en/roadmap.md [hunk 1]
```diff
@@ -37,36 +37,42 @@ The following GitOps Toolkit APIs are considered production ready:
...
-  - [x] [RFC-0002](https://github.com/fluxcd/flux2/tree/main/rfcs/0002-helm-oci) Flux OCI support for Helm
-  - [ ] [RFC](https://github.com/fluxcd/flux2/pull/2601) Flux OCI support for Kubernetes manifests
+- **OCI Artifacts**
+    - [x] [RFC-0002](https://github.com/fluxcd/flux2/tree/main/rfcs/0002-helm-oci) Flux OCI support for Helm
+    - [x] [RFC-0003](https://github.com/fluxcd/flux2/tree/main/rfcs/0003-kubernetes-oci) Flux OCI support for Kubernetes manifests
+    - [x] [End-to-end testing for OIDC auth with AWS, Azure and Google Cloud container registries](hhttps://github.com/fluxcd/pkg/tree/main/oci/tests/integration)
+    - [ ] [Verify OCI artifacts with cosign](https://github.com/fluxcd/source-controller/issues/863)
 
-- API consolidation ([fluxcd/flux2#1601](https://github.com/fluxcd/flux2/issues/1601))
+- **API consolidation** ([fluxcd/flux2#1601](https://github.com/fluxcd/flux2/issues/1601))
```
Comment (pjbgf) [inline content/en/roadmap.md:63]: It may be worth also adding an item around standardise validation at API level.

Comment (stefanprodan) [reply content/en/roadmap.md:63]: Added, thanks!

**Comment by pjbgf** [comment]
https://github.com/fluxcd/flux2/issues/1602#issuecomment-1131951114

File: content/en/roadmap.md [hunk 2]
```diff
@@ -37,37 +37,44 @@ The following GitOps Toolkit APIs are considered production ready:
...
     - [ ] Adopt Kubernetes [kstatus](https://github.com/kubernetes-sigs/cli-utils/tree/v0.25.0/pkg/kstatus#conditions) standard conditions
     - [ ] Standardize events and status conditions metadata
+    - [ ] [Standardize the OpenAPI validation](https://github.com/fluxcd/flux2/issues/2993) for the `toolkit.fluxcd.io` CRDs
 
-- Documentation improvements
+- **Git improvements** ([fluxcd/flux2#2593](https://github.com/fluxcd/flux2/issues/2593))
+    - [ ] [Consolidate Git implementations](https://github.com/fluxcd/pkg/issues/245)
+    - [ ] [End-to-end testing for Git protocols](https://github.com/fluxcd/source-controller/issues/656)
```
Comment (pjbgf) [inline content/en/roadmap.md:70]: The previous issue was around the Git Server requirements for our test needs. I created a new issue that covers test requirements.
```suggestion
    - [ ] [End-to-end testing for Git protocols](https://github.com/fluxcd/pkg/issues/334)
```

File: content/en/roadmap.md [hunk 3]
```diff
@@ -37,37 +37,44 @@ The following GitOps Toolkit APIs are considered production ready:
...
+    - [ ] [Verify OCI artifacts with cosign](https://github.com/fluxcd/source-controller/issues/863)
 
-- API consolidation ([fluxcd/flux2#1601](https://github.com/fluxcd/flux2/issues/1601))
+- **API consolidation** ([fluxcd/flux2#1601](https://github.com/fluxcd/flux2/issues/1601))
     - [ ] Adopt Kubernetes [kstatus](https://github.com/kubernetes-sigs/cli-utils/tree/v0.25.0/pkg/kstatus#conditions) standard conditions
     - [ ] Standardize events and status conditions metadata
+    - [ ] [Standardize the OpenAPI validation](https://github.com/fluxcd/flux2/issues/2993) for the `toolkit.fluxcd.io` CRDs
 
-- Documentation improvements
+- **Git improvements** ([fluxcd/flux2#2593](https://github.com/fluxcd/flux2/issues/2593))
```
Comment (pjbgf) [inline content/en/roadmap.md:68]: The old issue was focusing on a lot of tasks to get Git stable. I created a new one which is higher level:
```suggestion
- **Git improvements** ([fluxcd/flux2#3039](https://github.com/fluxcd/flux2/issues/3039))
```

File: content/en/roadmap.md [hunk 4]
```diff
@@ -37,37 +37,44 @@ The following GitOps Toolkit APIs are considered production ready:
...
+    - [ ] [Consolidate Git implementations](https://github.com/fluxcd/pkg/issues/245)
+    - [ ] [End-to-end testing for Git protocols](https://github.com/fluxcd/source-controller/issues/656)
+
+- **Documentation improvements**
     - [x] Consolidate the docs under [fluxcd.io](https://fluxcd.io) website
     - [x] Gather feedback on the [migration guides](https://github.com/fluxcd/flux2/discussions/413) and address more use-cases
     - [x] Cloud specific guides (AWS, Azure, Google Cloud)
```
Comment (pjbgf) [inline content/en/roadmap.md:75]: @stefanprodan Did we document Workload Identity for Azure Storage already? 
On the list of items that was the last outstanding item: https://github.com/fluxcd/website/issues/856

Comment (stefanprodan) [reply content/en/roadmap.md:75]: Workload Identity is an alpha feature, does anyone uses it in Azure?

Comment (pjbgf) [reply content/en/roadmap.md:75]: We have got it asked for a few times, and due to that added on https://github.com/fluxcd/website/issues/961.
I will remove that from GA and create an issue specific to it.

File: content/en/roadmap.md [hunk 5]
```diff
@@ -37,37 +37,44 @@ The following GitOps Toolkit APIs are considered production ready:
...
-    - [ ] Incident management and troubleshooting guides
-    - [ ] Developer guides for contributing to and extending Flux
+    - [x] Incident management and troubleshooting guides
```
Comment (pjbgf) [inline content/en/roadmap.md:76]: Shall we mark this issue as done as well?
https://github.com/fluxcd/website/issues/854

Comment (stefanprodan) [reply content/en/roadmap.md:76]: Yes please

**Pull request merged**